### PR TITLE
chore(docs): Old occurrences of gatbyjs.org

### PR DESCRIPTION
**Pull request by AnilSeervi**
## Description
- Fix instances of `gatsby.org`
- Typo: gatbyjs.org -> gatsbyjs.com
- fix 404 links

### CODE REVIEW

**Comment by tyhopp** [comment]
Hi @AnilSeervi, thanks for the PR!

It looks like the CI didn't run on your PR, could you push an empty commit or [enable us to make edits](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork) so that we can have it run?

I also had a look at the docs subdir and noticed there were other occurrences of `gatsbyjs.org` (such as [this occurrence](https://github.com/gatsbyjs/gatsby/blob/master/docs/contributing/gatsby-style-guide.md?plain=1#L45)) that we can change (ignoring store and example urls like using-gatsby-image). If you're interested, it would be great if you could help also change those.

**Comment by AnilSeervi** [comment]
> Hi @AnilSeervi, thanks for the PR!
> 
> It looks like the CI didn't run on your PR, could you push an empty commit or [enable us to make edits](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork) so that we can have it run?
> 
> I also had a look at the docs subdir and noticed there were other occurrences of `gatsbyjs.org` (such as [this occurrence](https://github.com/gatsbyjs/gatsby/blob/master/docs/contributing/gatsby-style-guide.md?plain=1#L45)) that we can change (ignoring store and example urls like using-gatsby-image). If you're interested, it would be great if you could help also change those.

I went ahead and fixed all occurences of `gatbsy.org` in _/docs_

Also fixed some 404 links while I was at it ✨

**Review by tyhopp — APPROVED** [review]
Looks great, thanks @AnilSeervi!

File: docs/docs/how-to/styling/using-web-fonts.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ Some examples of web font services include [Google Fonts](https://fonts.google.c
 
 ### How to use Web Font Loader with Typekit
 
-You can add Typekit Web Fonts to your project by using the [gatsby-plugin-web-font-loader](https://www.gatsbyjs.org/packages/gatsby-plugin-web-font-loader/?=font) and your [Adobe Fonts project id](https://fonts.adobe.com/my_fonts#web_projects-section). For example, this is how you can add Futura to your project.
+You can add Typekit Web Fonts to your project by using the [gatsby-plugin-web-font-loader](https://www.gatsbyjs.com/packages/gatsby-plugin-web-font-loader/?=font) and your [Adobe Fonts project id](https://fonts.adobe.com/my_fonts#web_projects-section). For example, this is how you can add Futura to your project.
```
Comment (tyhopp) [inline docs/docs/how-to/styling/using-web-fonts.md:21]:
```suggestion
You can add Typekit Web Fonts to your project by using the [gatsby-plugin-web-font-loader](/plugins/gatsby-plugin-web-font-loader/?=font) and your [Adobe Fonts project id](https://fonts.adobe.com/my_fonts#web_projects-section). For example, this is how you can add Futura to your project.
```

Links in docs should be relative to make use of `gatsby-link`.

**Review by tyhopp — CHANGES_REQUESTED** [review]
I did find one small thing, could you please use relative links for the case above? Thanks!

**Review by tyhopp — APPROVED** [review]

**Pull request merged**